common/djangoapps/util/milestones_helpers.py

```diff
@@ -408,3 +408,16 @@ def add_user_milestone(user, milestone):
         return None
     from milestones import api as milestones_api
     return milestones_api.add_user_milestone(user, milestone)
+
+
+def get_service():
+    """
+    Returns MilestonesService instance if feature flag enabled;
+    else returns None.
+
+    Note: MilestonesService only has access to the functions
+    explicitly requested in the MilestonesServices class
+    """
+    if not settings.FEATURES.get('MILESTONES_APP', False):
```

Comment:
So this means your PR also requires that the `MILESTONES_APP` feature flag be enabled on stage and prod.  Didn't that also get reverted yesterday?  @efischer19 @sanfordstudent ?